sdks/java/harness/src/test/java/org/apache/beam/fn/harness/BeamFnDataReadRunnerTest.java

```diff
@@ -103,6 +104,12 @@
           .setApiServiceDescriptor(Endpoints.ApiServiceDescriptor.getDefaultInstance())
           .setCoderId(CODER_SPEC_ID)
           .build();
+  private static final SplitResult mockSplitResult =
```

Comment:
nit: mockSplitResult -> TEST_SPLIT_RESULT

mock in a variable name usually means that it isn't a real instance of something.